### PR TITLE
Update scale-capacity.md

### PR DESCRIPTION
**Pull request by mihirwagleMSFT**
Updated known issues

### CODE REVIEW

**Comment by mihirwagleMSFT** [comment]
Kate:
Kesem is out. Can I get this known issue approved?
Thanks

Mihir Wagle

________________________________

File: docs/enterprise/scale-capacity.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ To scale your capacity you need:
 
 6. Confirm your tier by viewing the overview tab. The current pricing tier is listed.
 
+## Known issues
+
+Currently, when you scale a capacity from less than a F64 to an F64 or above, it may take up to 3 hours for this to be reflected in licensing verification. What this means is that Fabric Free users may see a request to upgrade to a Power BI Pro license when they try to access a Power BI report. We are working on a fix.
```
Comment (KesemSharabi) [inline docs/enterprise/scale-capacity.md:41]: Please change to:

'Scaling up a capacity that’s smaller than F64, may take up to three hours. During this time, Fabric Free users may see a request to upgrade to a Power BI Pro license when they try to access a Power BI report.'

We don't use time related wording (such as currently, now, previously, etc.) in technical documentation. Once the fix is out, we can remove thisstatement.

File: docs/enterprise/scale-capacity.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ To scale your capacity you need:
 
 6. Confirm your tier by viewing the overview tab. The current pricing tier is listed.
 
+## Known issues
```
Comment (KesemSharabi) [inline docs/enterprise/scale-capacity.md:39]: I suggest renaming this to 'Considerations and limitations'.

**Comment by Court72** [comment]
@mihirwaglemsft - Can you respond to the requested changes or let us know if you would like us to commit them? 

@KesemSharabi - After the changes are made, please add a `#sign-off` command to signal that the PR is ready to merge.

File: docs/enterprise/scale-capacity.md (hunk 2 above)
Comment (mihirwagleMSFT) [reply docs/enterprise/scale-capacity.md:39]: making these changes

**Comment by KesemSharabi** [comment]
#sign-off

**Pull request merged**